changelog.md

```diff
@@ -1,7 +1,7 @@
 # Changelog
 
 ## 0.1.0
-- Initial release
+Initial release
 
-## Unreleased
+## 0.2.0
```

Comment:
Adding `FileOpeningBuilder` is not a breaking change. Bump version to 0.1.1

Reply:
Addressed in: [Adapt version number](https://github.com/messerli-informatik-ag/file-system/pull/6/commits/86455dc14b3349e78815d2ed2eca384f8c8c3b65)